File: Java/3516. Find Closest Person.java
class Solution {
    public int findClosest(int x, int y, int z) {
        int diff1 = Math.abs(x - z), diff2 = Math.abs(y - z);
        if (diff1 < diff2) {
            return 1;
        } else if (diff1 > diff2) {
            return 2;
        } else {
            return 0;
        }
    }
}

File: Python3/3516. Find Closest Person.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        diff1 = abs(x - z)
        diff2 = abs(y - z)
        if diff1 < diff2:
            return 1
        elif diff1 > diff2:
            return 2
        else:
            return 0